components/support/sync15-traits/src/lib.rs

```diff
@@ -11,6 +12,7 @@ mod server_timestamp;
 mod store;
 pub mod telemetry;
 
+pub use bridged_engine::BridgedEngine;
```

Comment:
Can we export the results type, too, so that `golden_gate` can hold onto it and pass it back to the main thread?